deploy/mainnet/467_add_to_GeneralizedSwapMigrator.ts

```diff
@@ -0,0 +1,95 @@
+import { DeployFunction } from "hardhat-deploy/types"
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { CHAIN_ID } from "../../utils/network"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts, getChainId, ethers } = hre
+  const { execute, deploy, get, getOrNull, log, save } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  if ((await getChainId()) == CHAIN_ID.MAINNET) {
```

Comment:
+1 on Duy's comments. Should never use loose equality in JS. https://2ality.com/2011/12/strict-equality-exemptions.html